examples/models/phi-3-mini/main.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// main.cpp
+
+#include <iostream>
+
+#include <executorch/extension/evalue_util/print_evalue.h>
+#include <executorch/extension/module/module.h>
+#include <executorch/extension/runner_util/managed_tensor.h>
+#include <executorch/runtime/core/exec_aten/exec_aten.h>
+#include <executorch/runtime/core/exec_aten/util/dim_order_util.h>
+#include <executorch/runtime/core/exec_aten/util/tensor_util.h>
+
+#include "sentence_piece_tokenizer.h"
+
+using namespace torch::executor;
+
+// The value of the phi-3-mini `<|endoftext|>` token.
+#define ENDOFTEXT_TOKEN 32000
+#define VOCABULARY_SIZE 32064
+
+void generate(
```

Comment:
Is there any chance we can reuse the existing runner for phi 3?

Reply:
yes. We have a separate task created to refactor the code such that the `export_lib` and `runner` are shared. I will do that refactor after merging this.

Reply:
Please add a TODO here in case we forget